### ISSUE TITLE
[Update Request]: Roblox

### ISSUE THREAD

**Issue by Jayman2000** [microsoft/winget-pkgs#190435]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Roblox.Roblox

### Package Version

1.6.2.6410741

### Please describe the changes you would like to see

Steps to Reproduce:

1. Open an Administrator Command Prompt.

2. Run `winget install Roblox.Roblox`.

Result:

```
Microsoft Windows [Version 10.0.19045.5073]
(c) Microsoft Corporation. All rights reserved.

C:\Windows\system32>winget install Roblox.Roblox
Found Roblox Player [Roblox.Roblox] Version 1.6.2.6410741
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://setup.rbxcdn.com/RobloxPlayerLauncher.exe
  ██████████████████████████████  6.95 MB / 6.95 MB
Installer hash does not match; this cannot be overridden when running as admin

C:\Windows\system32>
```

**Comment by Jayman2000** [comment]
> If any of the above are duplicates, please consider closing this issue out and adding additional context in the original issue.

That issue has already been fixed. Should that old issue be reopened even though the installer has a different hash now?

**Comment by Dragon1573** [comment]
> That issue has already been fixed. Should that old issue be reopened even though the installer has a different hash now?

No, negative. They are caused by exactly the same reason, but are 2 unique issues. Just keep this issue open ❤

> [!IMPORTANT]
>
> This package seems using version-unrelated vanity URL (which always points to the latest version) for installer file downloading. This problem will keep exists, until we finally find a version-related static URL (which always point to exactly a specific version).

**Comment by Dragon1573** [comment]
> [!WARNING]
>
> - This package seems not write version info to _Windows Registry_.
> - This package seems using hash value as version number.
> - Failed to launch in _Windows Sandbox_.

![Image](https://github.com/user-attachments/assets/838977c7-457e-4eca-a923-6eb76406e33b)
![Image](https://github.com/user-attachments/assets/2ff26644-c5d9-49d8-b432-e43c6ae2f6eb)
